generic-collection.js

```diff
@@ -5,6 +5,8 @@ function GenericCollection() {
     throw new Error("Can't construct. GenericCollection is a mixin.");
 }
 
+GenericCollection.EmptyArray = Object.freeze([]);
```

Comment:
Thanks Kris, missed that, the problem is that there are many places where it should be used, (empty minus/plus) but these don’t require shim-array.js. Requiring it seems a bit wasteful and would increase the load time of collections a bit

> On Feb 10, 2015, at 5:41 PM, Kris Kowal <notifications@github.com> wrote:
> 
> array.js